.ddev/config.yaml

```diff
@@ -0,0 +1,160 @@
+APIVersion: v1.12.0-alpha2
+name: TYPO3-Console
+type: typo3
+docroot: .Build/public
+php_version: "7.2"
+webserver_type: apache-fpm
+router_http_port: "80"
+router_https_port: "443"
```

Comment:
I'd prefer these to be other ports, as these require root privs to open and might conflict with other running things on contributor's machines. Switching to non standard ports might help do reduce conflicts

Reply:
That should be fine. And with the new command `ddev launch` it's not an issue to access the correct ports automatically with the browser. Will change it, any port wishes? If not I'll take 8080 and 8443...

Reply:
With [quicksprint](https://github.com/drud/quicksprint) we use 8080 and 8443 as suggested. Those are easy to remember and usually don't conflict.